### PR TITLE
feat: experimental web support

### PR DESCRIPTION
**Pull request by toddbaert**
Signed-off-by: Todd Baert <toddbaert@gmail.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,7 +40,10 @@
       "import": "./dist/esm/index.js",
       "require": "./dist/cjs/index.js",
       "default": "./dist/esm/index.js"
-    }
+    },
+    "import": "./dist/esm/index.js",
+    "require": "./dist/cjs/index.js",
+    "default": "./dist/esm/index.js"
```
Comment (agentgonzo) [inline package.json:46]: We can probable remove the node-specific exports above given they are exactly the same as these new ones.

Comment (toddbaert) [reply package.json]: You're right!

**Comment by toddbaert** [comment]
@james-milligan I've published another experimental version. It shouldn't have any breaking changes, but if you could confirm next week that it works as expected with your web provider, I'd appreciate it!

`@openfeature/js-sdk@0.4.0-experimental-e9ba70956ed77c7b41032b149a8f6bf915ae50e4`

**Comment by james-milligan** [comment]
@toddbaert i have this version working with the experimental version of the web provider

**Comment by agentgonzo** [comment]
I have a fully working web application using the cloudbees feature provider and this experimental release 🎉
screen-recording

**Review by agentgonzo — APPROVED** [review]

**Review by beeme1mr — APPROVED** [review]

**Pull request merged**